yarn.lock

```diff
@@ -1486,7 +1495,12 @@
   dependencies:
     "@types/node" "*"
 
-"@types/node@*", "@types/node@>=6":
+"@types/node@*":
+  version "13.9.0"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-13.9.0.tgz#5b6ee7a77faacddd7de719017d0bc12f52f81589"
+  integrity sha512-0ARSQootUG1RljH2HncpsY2TJBfGQIKOOi7kxzUY6z54ePu/ZD+wJA8zI2Q6v8rol2qpG/rvqsReco8zNMPvhQ==
+
+"@types/node@>=6":
```

Comment:
Same here, I think you can dedupe.

Reply:
<img width="921" alt="Screen Shot 2020-03-10 at 2 59 10 PM" src="https://user-images.githubusercontent.com/308886/76349088-b9a44100-62df-11ea-8aed-5de419eb9a27.png">

I'm concerned about bumping it from 12 to 13. Maybe an apollo memory update would be better in another pr?